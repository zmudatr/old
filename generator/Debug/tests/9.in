USTAW S ZXUH*
USTAW X Vop1
SKOPIUJ S X
WYPISZ S
WYPISZ X
ZERUJ S
ZERUJ X
USTAW Z lYiwi9A
USTAW K mJm8IiQAG
SKOPIUJ Z K
WYPISZ Z
WYPISZ K
ZERUJ Z
ZERUJ K
USTAW F H5NV8ub_y*
USTAW L _KAJ*B
SKOPIUJ F L
WYPISZ F
WYPISZ L
ZERUJ F
ZERUJ L
USTAW S Ve
USTAW P Qv4pp
SKOPIUJ S P
WYPISZ S
WYPISZ P
ZERUJ S
ZERUJ P
USTAW S +Ftsopcy6
USTAW V Jb/-YN8+
SKOPIUJ S V
WYPISZ S
WYPISZ V
ZERUJ S
ZERUJ V
USTAW T KU4hAl
USTAW P 76fFU1x
SKOPIUJ T P
WYPISZ T
WYPISZ P
ZERUJ T
ZERUJ P
USTAW I 3-Kr/r3
USTAW S -
SKOPIUJ I S
WYPISZ I
WYPISZ S
ZERUJ I
ZERUJ S
USTAW Y G
USTAW D Ar
SKOPIUJ Y D
WYPISZ Y
WYPISZ D
ZERUJ Y
ZERUJ D
USTAW X -5t
USTAW C DyHLDkPa4H
SKOPIUJ X C
WYPISZ X
WYPISZ C
ZERUJ X
ZERUJ C
USTAW B X1Kh-IwkX
USTAW W 7kWR-M
SKOPIUJ B W
WYPISZ B
WYPISZ W
ZERUJ B
ZERUJ W
USTAW T pi8wtG/S
USTAW L kJzbhM3
SKOPIUJ T L
WYPISZ T
WYPISZ L
ZERUJ T
ZERUJ L
USTAW A i
USTAW L ySb
SKOPIUJ A L
WYPISZ A
WYPISZ L
ZERUJ A
ZERUJ L
USTAW Q HHE1BTZ9s
USTAW U e
SKOPIUJ Q U
WYPISZ Q
WYPISZ U
ZERUJ Q
ZERUJ U
USTAW Q EGtda
USTAW P -3C2F
SKOPIUJ Q P
WYPISZ Q
WYPISZ P
ZERUJ Q
ZERUJ P
USTAW I GBFNvEc
USTAW T uK8ro
SKOPIUJ I T
WYPISZ I
WYPISZ T
ZERUJ I
ZERUJ T
USTAW S A0
USTAW Z -
SKOPIUJ S Z
WYPISZ S
WYPISZ Z
ZERUJ S
ZERUJ Z
USTAW U Upv8q4P9*j
USTAW Y jxcpx
SKOPIUJ U Y
WYPISZ U
WYPISZ Y
ZERUJ U
ZERUJ Y
USTAW U q6r31rXZNl
USTAW F g
SKOPIUJ U F
WYPISZ U
WYPISZ F
ZERUJ U
ZERUJ F
USTAW F eJb4OqgyYS
USTAW B 2CTVCegE
SKOPIUJ F B
WYPISZ F
WYPISZ B
ZERUJ F
ZERUJ B
USTAW L /_totfjb*V
USTAW Q Ao+-bQ*
SKOPIUJ L Q
WYPISZ L
WYPISZ Q
ZERUJ L
ZERUJ Q
