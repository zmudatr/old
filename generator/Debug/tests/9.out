ZXUH*Vop1
Vop1
lYiwi9AmJm8IiQAG
mJm8IiQAG
H5NV8ub_y*_KAJ*B
_KAJ*B
VeQv4pp
Qv4pp
+Ftsopcy6Jb/-YN8+
Jb/-YN8+
KU4hAl76fFU1x
76fFU1x
3-Kr/r3-
-
GAr
Ar
-5tDyHLDkPa4H
DyHLDkPa4H
X1Kh-IwkX7kWR-M
7kWR-M
pi8wtG/SkJzbhM3
kJzbhM3
iySb
ySb
HHE1BTZ9se
e
EGtda-3C2F
-3C2F
GBFNvEcuK8ro
uK8ro
A0-
-
Upv8q4P9*jjxcpx
jxcpx
q6r31rXZNlg
g
eJb4OqgyYS2CTVCegE
2CTVCegE
/_totfjb*VAo+-bQ*
Ao+-bQ*
